Core/include/Acts/Geometry/Layer.hpp

```diff
@@ -281,9 +282,12 @@ class Layer : public virtual GeometryObject {
   /// @param layerID is the geometry id of the volume
   ///                as calculated by the TrackingGeometry
   /// @param hook Identifier hook to be applied to surfaces
+  /// @param logger A @c LoggerWrapper instance
+  ///
   void closeGeometry(const IMaterialDecorator* materialDecorator,
                      const GeometryIdentifier& layerID,
-                     const GeometryIdentifierHook& hook);
+                     const GeometryIdentifierHook& hook,
+                     LoggerWrapper logger = getDummyLogger());
```

Comment:
Could this switch to using `const Logger&` with the same `getDummyLogger()` default? I'm trying to phase out `LoggerWrapper`.

Reply:
Sure. Will do.